adhocracy-plus/config/settings/dev.py

```diff
@@ -14,7 +14,10 @@
 except ImportError:
     pass
 else:
-    INSTALLED_APPS += ("debug_toolbar",)
+    INSTALLED_APPS += (
+        "allauth.socialaccount.providers.dummy",
```

Comment:
this ties the dummy provider to whether the debug_toolbar is installed or not, wouldn't it be a bit clearer to have a separate `INSTALLED_APPS += (""allauth.socialaccount.providers.dummy")` somehwere outside the try block?